Assignment0/matrixMatrixProduct.cpp

```diff
@@ -0,0 +1,32 @@
+#include "header.hpp"
+#include <iostream>
+#include <vector>
+
+using namespace std;
```

Comment:
It is suggested not to include a namespace but rather to use scope, for example, `std::cout`.